packages/money_transfer_service/db_utils/index.js

```diff
@@ -0,0 +1,8 @@
+const mongoose = require('mongoose');
+const MoneyTransferSchema  = require('./Models/MoneyTransfer');
+
+module.exports = config => {
+    mongoose.Promise = global.Promise;
+    mongoose.connect(config.mongoURI);
+    mongoose.model('tMoneyTransfer', MoneyTransferSchema,'tMoneyTransfer');
```

Comment:
```suggestion
    mongoose.model('tMoneyTransfer', MoneyTransferSchema, 'tMoneyTransfer');
```
tMoneyTransfer is the table name right?